src/makeCalculator.js

```diff
@@ -4,7 +4,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(currentResult, num) {
```

Comment:
The `add` method takes `currentResult` as a parameter, but this is unnecessary since `result` is already maintained internally. Consider using `result` directly within the method.